src/data.c

```diff
@@ -423,7 +423,7 @@ static void print_kv_data(data_printer_context_t *printer_ctx, data_t *data, cha
 			}
 		}
 		if (!strcmp(data->key, "time"))
-                    fprintf(file, "");
+                    ;
```

Comment:
Change this to a commented fprintf statement. Otherwise the code looks very peculiar.